src/main/resources/springintegration/flows.xml

```diff
@@ -9,6 +9,7 @@
     <int:annotation-config/>
 
     <import resource="case-gateway-event-inbound.xml"/>
```

Comment:
consistent naming here please

Reply:
Names of 3 included files renamed. 
Please review.